### PR TITLE
fix(xwf): remove xwf reference from installation and deploy scripts

### PR DESCRIPTION
**Pull request by crasu**
## Summary

Remove xwf references in scripts

## Test Plan

Tested if installation still runs 

Relates to #13546

### CODE REVIEW

File: bazel/scripts/check_py_bazel.sh [hunk 1]
```diff
@@ -45,7 +45,6 @@ DENY_LIST_NOT_RELEVANT=(
   "./protos"
   "./show-tech"
   "./third_party"
-  "./xwf/gateway/deploy"
```
Comment (crasu) [inline bazel/scripts/check_py_bazel.sh]: can we remove this already before #13545 is merged @LKreutzer

Comment (LKreutzer) [reply bazel/scripts/check_py_bazel.sh]: Removing the line while the folder still exists should lead to failing bazel workflows on PRs. So we should keep it and remove it in #13545.

Comment (crasu) [reply bazel/scripts/check_py_bazel.sh]: ok moved it there

**Comment by crasu** [comment]
@alexzurbonsen  Could you take over this pr?

**Review by sebathomas — APPROVED** [review]

**Comment by alexzurbonsen** [comment]
Changes are now part of https://github.com/magma/magma/pull/13570.

**Pull request closed without merging**